packages/core/src/internal/io.js

```diff
@@ -141,9 +141,10 @@ export function call(fnDescriptor, ...args) {
     check(
       fnDescriptor,
       isNotDelayEffect,
-      `instead of writing \`yield call(delay, ${
-        args[0]
-      })\` where delay is an effect from \`redux-saga/effects\` you should write \`yield delay(${args[0]})\``,
+      () =>
+        `instead of writing \`yield call(delay, ${
+          args[0]
```

Comment:
this will still throw unhelpful error for
```js
yield call(delay, Symbol())
```

IMHO better solution would be to just stringify explicitly args[0] - `String(args[0])`

Reply:
Ah, didn't think about that. I'll change.